config/connection.js

```diff
@@ -1,13 +1,13 @@
-const Sequelize = require('sequelize');
-require('dotenv').config();
+const Sequelize = require("sequelize");
+require("dotenv").config();
 
 const sequelize = new Sequelize(
   process.env.DB_NAME,
   process.env.DB_USER,
   process.env.DB_PASSWORD,
   {
-    host: 'localhost',
-    dialect: 'mysql',
+    host: "localhost",
```

Comment:
I need to change this to "127.0.0.1" for it to run on my machine--does that still run for you on Windows?

Reply:
Nevermind, Zack just recommended we don't change this in code! Lo and I will just keep local changes to ourselves.